src/App.css

```diff
@@ -1,65 +1,305 @@
+@import url('https://fonts.googleapis.com/css2?family=Gothic+A1:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Slab:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
[optional] Good job✔️ for adding the suggested font family. However,  Instead of using @import to load fonts, it's generally better to include the fonts directly in your HTML using the `<link>` tag in the `<head>` section. This helps in reducing additional HTTP requests and improves page loading speed.🤔🤔